### DISCUSSION TITLE
Nintendo Switch

### DISCUSSION THREAD

**Discussion by parkuman** [raphamorim/rio#130] [General]
Hey! 

Just found this project and can't wait to try it out once I'm home. I noticed in the readme the mention of trying to get it working on a Switch. I love this idea and just recently got into modding my Switch. Would love to hear any updates on how this pet project is going!

**Comment by raphamorim** [comment]
Hey there @parkuman 
Thanks for the message.

That’s great! I plan to start work on that once the WebAssembly version has been fully ported. Looking forward to work on it :)
